packages/cli/rollup.config.js

```diff
@@ -0,0 +1,41 @@
+import esbuild from 'rollup-plugin-esbuild'
+import resolve from '@rollup/plugin-node-resolve'
+import commonjs from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json'
+import pkg from './package.json'
+
+const entries = [
+  'src/index.ts'
+]
+
+const external = [
+  ...Object.keys(pkg.dependencies),
+  ...Object.keys(pkg.peerDependencies),
+]
+
+export default ({ watch }) => [
+  {
+      input: entries,
+      output: {
+        dir: 'dist',
+        format: 'esm',
+        sourcemap: 'inline',
+      },
+      external,
+      plugins: [
+        resolve({
+          preferBuiltins: true,
+        }),
+        json(),
+        commonjs(),
+        esbuild({
+          target: 'node14',
```

Comment:
`esbuild` is a Javascript transpiler that's faster than [Babel](https://babeljs.io). Through `target` we are telling `esbuild` what's the minimum version of Node that we are supporting.